src/cartridges/int_adyen_SFRA/cartridge/adyen/scripts/donations/adyenGiving.js

```diff
@@ -26,12 +26,14 @@ const AdyenHelper = require('*/cartridge/adyen/utils/adyenHelper');
 const AdyenConfigs = require('*/cartridge/adyen/utils/adyenConfigs');
 const constants = require('*/cartridge/adyen/config/constants');
 const AdyenLogs = require('*/cartridge/adyen/logs/adyenCustomLogs');
+const setErrorType = require('*/cartridge/adyen/logs/setErrorType');
+const { AdyenError } = require('*/cartridge/adyen/logs/adyenError');
 
 // eslint-disable-next-line complexity
 function donate(donationReference, donationAmount, orderToken) {
   try {
```

Comment:
Should we remove try/catch here, considering we are adding it at main function `donation()`?